NuGet.Config

```diff
@@ -1,8 +1,9 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<configuration>

```

Comment:
why your pr changes so many lines? different encoding?

Reply:
using the `LF` encoding. that is set in `.editorconfig`